SPECS/meson/meson.spec

```diff
@@ -1,21 +1,28 @@
 Summary:        Extremely fast and user friendly build system
 Name:           meson
 Version:        0.60.2
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Looks like you need to bump the toolchain manifest for this package, by the way.